### PR TITLE
fix shared update

### PR DESCRIPTION
**Pull request by hakos47**
It's a temporary fix while we see how to do it better. but this in principle fixes the error that when a client entered it did not get the table of items etc updated. since shared_scripts does not serve to update information on both sides (client, server) it would fix problems with addItems, addItem, etc.

It is not the best solution. but it's a temporary fix

## Checklist

- [ x] I have personally loaded this code into an updated qbcore project and checked all of its functionality.
- [ x] My code fits the style guidelines.
- [ x] My PR fits the contribution guidelines.

### CODE REVIEW

**Comment by amantu-qbit** [comment]
Great work

**Comment by amantu-qbit** [comment]
Also, why don't just push the whole QBshared table to client?

**Comment by hakos47** [comment]
> Also, why don't just push the whole QBshared table to client?

The idea is good, but I think we should review it in more detail, as it could be an excessive burden. We should consider more efficient alternatives.

**Comment by hakos47** [comment]
> Also, why don't just push the whole QBshared table to client?

I've changed the system and made it receive the information upon connection. I believe it works better than the previous one, at least in the tests I've conducted, and it fixes all the problems with job updates, vehicles, etc.

**Comment by sirhvd** [comment]
I think you forgot to add "**src**" when using TriggerClientEvent.
`TriggerClientEvent('QBCore:Client:SharedUpdate', src, QBCore.Shared)`

File: server/events.lua [hunk 1]
```diff
@@ -99,6 +99,7 @@ local function onPlayerConnecting(name, _, deferrals)
     end
 
     -- Add any additional defferals you may need!
+    TriggerClientEvent('QBCore:Client:SharedUpdate', QBCore.Shared)
```
Comment (GhzGarage) [inline server/events.lua:102]: Missing source

**Pull request merged**